browser/extensions/api/brave_shields_api.cc

```diff
@@ -70,6 +85,8 @@ BraveShieldsSetBraveShieldsEnabledFunction::Run() {
   Profile* profile = Profile::FromBrowserContext(browser_context());
   ::brave_shields::SetBraveShieldsEnabled(profile, params->enabled, url);
 
+  RecordShieldsToggled();
```

Comment:
this is definitely not the right place to put this because it won't be captured on android. I think it should go in `brave_shields:: SetBraveShieldsEnabled `

Reply:
fixed